data/korath/korath missions.txt

```diff
@@ -577,3 +577,599 @@ mission "Efreti: Distress Call 8"
+			`	"Jenny Jetmaster," she says, with a choreographed bow. "My family used to do air shows all across human space, until my uncle got on the bad side of the Syndicate. We flew all the way from Bourne to Tinker to pick him up and smuggled him out here to New Tortuga where the Syndicate cannot reach us. Now I smuggle cargo through hyperspace in a 20th century aircraft!"`
+			branch questions
+				has "Korath [F01]: A Fishy Deal: down"
+			label down
+			action
+				set "Korath [F01]: A Fishy Deal: down"
+			`	Her face turns sour, and she says, "Unfortunately, I can't finish my current delivery. My mother is quite sick, and I want to get to Freedom before..." she trails off. "Well, a client really needs this delivery, and I have to find someone to do it."`
+				goto questions
+			label mission
+			action
+				set "Korath [F01]: A Fishy Deal: mission"
+			`	"It's the sort of delivery where you don't ask what sort of delivery it is. Take <tons> of fish to <planet>, and fly back here with <tons> of supplies. It's just two systems away, and the payment is 140,000 credits. Couldn't be easier."`
+				goto questions
+			label yup
+			action
+				log "Agreed to help a woman named Jenny Jetmaster who couldn't make a delivery. Apparently the delivery is illegal, though it's unlikely to be detected since it's in a region of space not patrolled by any law-abiding governments."
+			`	"Really? Okay, hold on..." looking down, she fidgets with a datapad, perhaps transferring permission for the cargo mission. "Yeah, okay." She looks back up at last. "Transfer is approved." Her crewman starts offloading the fish, preparing to transfer it to your ship.`
+			`	"If the authorities won't let you land, transmit them the serial number from the cargo container, and they'll open a bay for you. Oh, also, this is an illegal mission, so don't get caught, or it'll cost you. Well, it's unlikely anyone out here will care about laws, but I do believe in full disclosure."`
+			label "d'ya want the job"
+			choice
+				`	"I'm not afraid of dodging the cops... where there are none. Give me the cargo, and it'll reach its destination. You have my word."`
+					accept
+				`	"On second thought, I don't think I want this mission. Illegal fish deliveries are too strange for my tastes."`
+					decline
+				`	"If I'm just delivering fish, why is it illegal?"`
+					to display
+						not "Korath [F01]: A Fishy Deal: spicy"
+			label "'cause they're spicy"
+			action
+				set "Korath [F01]: A Fishy Deal: spicy"
+			`	"Nothing is wrong with the fish; the other end of the delivery is the problem, so the whole trade arrangement is illegal. There's all sorts of barrels and boxes that come back from <planet>. It's mostly mundane stuff; never any drugs, counterfeits, or weapons. Why the Republic would ban this... I haven't the slightest notion.`
```

Comment:
Because it's not "humans have found a way to trade with aliens instead of shooting them,"  but, from the Republic's point of view, "the Syndicate is telling us over an extremely fine dinner that this illegitimate world is actively collaborating with some some dangerous group that has been attacking merchants."

Reply:
The republic is widely known for being heavy-handed in terms of regulations. It's entirely possible that, since these particular fish have never been studied and their long-term effects analyzed for effects on people, that they have been deemed illegal. And being a potential threat to the established food providing industry, inspectors and researchers to do that analysis for some reason *always* have sixty other things they need to test first before getting to this. Assuming that anyone's paid the multi-million credit fee to get it onto their task list to begin with. Which no-one may even know is what they should be doing, since the above interests have worked hard to ensure that the inspectors aren't talking to them.

Reply:
That is a much better explanation than mine! Well done.

Reply:
So basically, there are many different groups who have their own reasons to want to restrict trade:
- Paradise worlds don’t want the Republic to support trade with a group that attacks merchants
- Syndicate wants to protect their monopoly 
- Deep doesn’t want to support trade using fish that hasn’t been proven safe
- Free Worlds is a new government that is either fighting a war or just fought one and doesn’t want to lose it all over something as trivial as fish
- Hai have a deal with the Republic, plus they still remember their own war with the Korath
- Something something Remnant lore
- Cap’n Pester is upset because the Korath haven’t purchased a license to play endless sky
etc.

Reply:
Yeah, ideally, the fish shipment would only be illegal in the Syndicate and Republic (which includes Deep and Paradise.)  The Hai getting upset is a stretch and the Free Worlds probably shouldn't care at all, but such nuance will have to wait until we can make government distinctions regarding legality.